### ISSUE TITLE
There is no way to opt out cache via Request Headers

### ISSUE THREAD

**Issue by daguitosama** [litespeedtech/lscache_wp#732]
Hi, I recently hit an issue with LittleSpeed Cache. It was overwriting a REST API call response, and giving back a cache version even if the request was made with headers asking for a no cache version like: **Cache-Control: "no-cache"** and **Pragma: "no-cache"**. I fix it by disabling REST Cache on settings of the plugin, but wanted to ask if this is the intended behaviour.

**Comment by timotei-litespeed** [comment]
@daguitosama 
There is this setting: https://docs.litespeedtech.com/lscache/lscwp/cache/#cache-rest-api Will it help if you disable this setting?

**Comment by seb86** [comment]
Yes, disabling helps but I think what we would like is a little more control on which REST API's that we cache or detection of the header conditions a REST API endpoint sets as to prevent caching if need be.

**Comment by timotei-litespeed** [comment]
There is this setting: https://docs.litespeedtech.com/lscache/lscwp/cache/#do-not-cache-uris
Also under there are other options that can help
